### ISSUE TITLE
Gradients handling

### ISSUE THREAD

**Issue by GalKepler** [GalBenZvi/dwiprep#17]
* DWIPrep version: 0.1.0
* Python version:3.9
* Operating System:Linux

### Description

It must be certain that all linear and non-linear transformation performed on the DWI data is also reflected in the corresponding B vector and values.
I saw that a similar issue was already referred to [here](https://community.mrtrix.org/t/rotating-bvecs-after-correction-for-susceptibility-induced-distortions-using-t1/2718/2), so it should be easy to manage.

Basically what we need to do is convert FSL or ANTs' transformation matrices into mrtirx3's format and than apply it to the DWI data in .mif format, since the above-mention thread declares that mrtrix deals with such transformations so it should be safe as long as we keep it in mrtrix's playground (.mif files).
